private/pkg/transport/http/httpserver/zap.go

```diff
@@ -22,17 +22,28 @@ import (
 	"go.uber.org/zap"
 )
 
-func newZapMiddleware(logger *zap.Logger) func(http.Handler) http.Handler {
+func newZapMiddleware(logger *zap.Logger, silentEndpoints []string) func(http.Handler) http.Handler {
 	return func(next http.Handler) http.Handler {
 		return http.HandlerFunc(func(writer http.ResponseWriter, request *http.Request) {
 			logEntry := newLogEntry(logger, request)
 			wrapResponseWriter := middleware.NewWrapResponseWriter(newCheckedResponseWriter(logger, writer, request), request.ProtoMajor)
-			defer logRequest(logEntry, wrapResponseWriter, time.Now())
+			if !contains(silentEndpoints, request.URL.Path) {
+				defer logRequest(logEntry, wrapResponseWriter, time.Now())
+			}
 			next.ServeHTTP(wrapResponseWriter, middleware.WithLogEntry(request, logEntry))
 		})
 	}
 }
 
+func contains(l []string, s string) bool {
```

Comment:
I was unsure about creating a dependency on https://pkg.go.dev/golang.org/x/exp/slices#Contains - perhaps it would it be better to convert to a map?

Reply:
Ye, I'd just `map[string]struct{}` and inline the `request.URL.Path` check.